project/Settings.scala

```diff
@@ -0,0 +1,65 @@
+import Dependencies._
+import Merging._
+import Testing._
+import Compiling._
+import sbt.Keys._
+import sbt._
+import sbtassembly.AssemblyPlugin.autoImport._
+
+object Settings {
+
+  val artifactory = "https://artifactory.broadinstitute.org/artifactory/"
+
+  val commonResolvers = List(
+    "artifactory-releases" at artifactory + "libs-release",
+    "artifactory-snapshots" at artifactory + "libs-snapshot"
+  )
+
+  val buildSettings: Seq[_root_.sbt.Def.Setting[_]] = Defaults.defaultSettings ++ Seq(
```

Comment:
I don't know what it does and I don't know what it is I want, so I'm afraid to touch it.